src/sentry/static/sentry/app/views/settings/project/projectOwnership/index.tsx

```diff
@@ -9,28 +10,27 @@ import Form from 'app/views/settings/components/forms/form';
 import JsonForm from 'app/views/settings/components/forms/jsonForm';
 import OwnerInput from 'app/views/settings/project/projectOwnership/ownerInput';
 import PermissionAlert from 'app/views/settings/project/permissionAlert';
-import SentryTypes from 'app/sentryTypes';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
 import TextBlock from 'app/views/settings/components/text/textBlock';
 import Button from 'app/components/button';
+import {Organization, Project} from 'app/types';
 
-const CodeBlock = styled('pre')`
-  word-break: break-all;
-  white-space: pre-wrap;
-`;
+type Props = {
+  organization: Organization;
+  project: Project;
+} & RouteComponentProps<{}, {}>;
```

Comment:
Are there any route parameters? Perhaps organization/project slug?

Reply:
There are but they are unused.

Reply:
I'm wondering if we should type them anyway. 
My reasoning is that they are there and therefore should be typed. When someone goes into that file in the future it will be hard for them to figure out what params we have access to.
What do you think?